Traceback (most recent call last):
  File "/Users/jasonpbu/Desktop/GTM_SDC_developing/tvt_monitor/monitoring.py", line 44, in <module>
    plt.imshow()
  File "/Users/jasonpbu/miniconda3/envs/spyder-env/lib/python3.9/site-packages/matplotlib/_api/deprecation.py", line 456, in wrapper
    return func(*args, **kwargs)
TypeError: imshow() missing 1 required positional argument: 'X'
/Users/jasonpbu/miniconda3/envs/spyder-env/lib/python3.9/site-packages/numpy/core/fromnumeric.py:3162: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
  return asarray(a).ndim
/Users/jasonpbu/miniconda3/envs/spyder-env/lib/python3.9/site-packages/numpy/lib/index_tricks.py:394: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray.
  newobj = array(item, copy=False, subok=True, ndmin=ndmin)
Traceback (most recent call last):
  File "/Users/jasonpbu/miniconda3/envs/spyder-env/lib/python3.9/site-packages/pandas/core/series.py", line 185, in wrapper
    raise TypeError(f"cannot convert the series to {converter}")
TypeError: cannot convert the series to <class 'float'>

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/jasonpbu/Desktop/GTM_SDC_developing/tvt_monitor/monitoring.py", line 78, in <module>
    sc.set_offsets(np.c_[x,y])
  File "/Users/jasonpbu/miniconda3/envs/spyder-env/lib/python3.9/site-packages/matplotlib/collections.py", line 566, in set_offsets
    (np.asarray(self.convert_xunits(offsets[:, 0]), 'float'),
ValueError: setting an array element with a sequence.
[22;0t]0;IPython: GTM_SDC_developing/tvt_monitor[22;0t]0;IPython: GTM_SDC_developing/tvt_monitor[22;0t]0;IPython: GTM_SDC_developing/tvt_monitor[0;31m---------------------------------------------------------------------------[0m
[0;31mIndexError[0m                                Traceback (most recent call last)
[0;32m~/Desktop/GTM_SDC_developing/tvt_monitor/monitoring.ipy[0m in [0;36m<module>[0;34m[0m
[1;32m     37[0m [0mfigure[0m[0;34m,[0m [0max[0m [0;34m=[0m [0mplt[0m[0;34m.[0m[0msubplots[0m[0;34m([0m[0;36m1[0m[0;34m,[0m [0;36m2[0m[0;34m,[0m [0mfigsize[0m[0;34m=[0m[0;34m([0m[0;36m10[0m[0;34m,[0m [0;36m8[0m[0;34m)[0m[0;34m)[0m [0;31m# 1 * 2 subplots[0m[0;34m[0m[0;34m[0m[0m
[1;32m     38[0m [0mx[0m[0;34m,[0m [0my[0m [0;34m=[0m [0;34m[[0m[0;34m][0m[0;34m,[0m [0;34m[[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 39[0;31m [0mline00[0m[0;34m,[0m [0;34m=[0m [0max[0m[0;34m[[0m[0;36m0[0m[0;34m,[0m [0;36m0[0m[0;34m][0m[0;34m.[0m[0mplot[0m[0;34m([0m[0mx[0m[0;34m,[0m [0my[0m[0;34m,[0m [0mmarker[0m[0;34m=[0m[0;34m"o"[0m[0;34m,[0m [0mls[0m[0;34m=[0m[0;34m""[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     40[0m [0mline01[0m[0;34m,[0m [0;34m=[0m [0max[0m[0;34m[[0m[0;36m0[0m[0;34m,[0m [0;36m1[0m[0;34m][0m[0;34m.[0m[0mplot[0m[0;34m([0m[0mx[0m[0;34m,[0m [0my[0m[0;34m,[0m [0mmarker[0m[0;34m=[0m[0;34m"o"[0m[0;34m,[0m [0mls[0m[0;34m=[0m[0;34m""[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m     41[0m [0;34m[0m[0m

[0;31mIndexError[0m: too many indices for array: array is 1-dimensional, but 2 were indexed
[22;0t]0;IPython: GTM_SDC_developing/tvt_monitor[22;0t]0;IPython: GTM_SDC_developing/tvt_monitor[22;0t]0;IPython: GTM_SDC_developing/tvt_monitor[22;0t]0;IPython: GTM_SDC_developing/tvt_monitor[22;0t]0;IPython: GTM_SDC_developing/tvt_monitor[22;0t]0;IPython: GTM_SDC_developing/tvt_monitor[22;0t]0;IPython: GTM_SDC_developing/tvt_monitor[22;0t]0;IPython: GTM_SDC_developing/tvt_monitor